typescript/processors/linkInheritedDocs.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Processor that links the inherited symbols to the associating dgeni doc.
+ **/
+module.exports = function linkInheritedDocs(typescriptSymbolMap) {
+  return {
+    $runAfter: ['readTypeScriptModules'],
```

Comment:
it is probably worth putting in a `$runBefore` property too, since we can't guarantee that it will be run before other processors otherwise...